<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>1300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bus and Routes Assignment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="assignbusroute">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>761</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Assign Bus and Route to driver</string>
    </property>
    <widget class="QTableWidget" name="tabelViewAssign">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>400</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="viewasslabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>380</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>View Assignments</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="busdriverassign">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>291</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Bus Driver Assignments</string>
     </property>
     <widget class="QTableWidget" name="busdrivertable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>271</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="buscomboBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="budIDlabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bus ID</string>
      </property>
     </widget>
     <widget class="QComboBox" name="drivercomboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="driveridlabel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Driver ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletebtnbusdriver">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="assignbtnbusdriver">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Assign</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="busrouteassign">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>291</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Bus Route Assignments</string>
     </property>
     <widget class="QTableWidget" name="busroutetable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>271</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="buscomboBox_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="budIDlabel_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bus ID</string>
      </property>
     </widget>
     <widget class="QComboBox" name="routecomboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deletebtnbusroute">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="assignbtnbusroute">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Assign</string>
      </property>
     </widget>
     <widget class="QLabel" name="routeidlabel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Route ID</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="dashboardbtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
